core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -57,9 +57,9 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
   <div>
     <background-banner></background-banner>
     <conversation-skin></conversation-skin>
-    {% if not iframed %}
-      <attribution-guide></attribution-guide>
-    {% endif %}
+
+    <attribution-guide ng-if="!iframed"></attribution-guide>
```

Comment:
`iframed` is defined in Base.js on line number 26. Since, exploration_player/exploration_player.html is extended from base.html, so I guess, it is defined there.

Reply:
That's great! :) 
This part looks good to me!